### PR TITLE
Add dashboard

### PR DESCRIPTION
**Pull request by bennetbo**
This adds a dashboard to the app, where users will be able to see their observation requests. This is also where we expose links to other functionality, like logging out or creating observation requests.

image

### CODE REVIEW

**Comment by bennetbo** [comment]
There is not a "clear" stopping point for when this is completed. Right now all the added functionality should work and existing functionality should still work/be as broken as before.

I would like to get my changes in now, so that everyone can keep iterating with the new approach (and get rid of the demo code which we had before).
A couple follow ups include:
- Come up with a new user invitation view, which removes the old `generate_invitation.html
- Redesign the `register_from_invitation.html` similar to what we have for login
- Actually write new observation request to the database
- Add an overview of submitted observations

File: authentication/views.py [hunk 1]
```diff
@@ -52,7 +55,7 @@ def login_user(request):
             request, form=LoginForm(), error="Invalid username or password"
         )
     auth.login(request, user)
-    return redirect("index")
+    return redirect("dashboard")
```
Comment (btw04) [inline authentication/views.py:58]: This could also be an entry in `settings.py` IMO

File: authentication/views.py [hunk 2]
```diff
@@ -81,39 +84,43 @@ def generate_user_invitation(request):
...
+        )
 
     invitation.delete()
     user = User.objects.create_user(username=invitation.email, email=invitation.email)
     user.set_password(form.cleaned_data["new_password1"])
     user.save()
     invitation.save()
     auth.login(request, user)
-    return redirect("index")
+    return redirect("dashboard")
```
Comment (btw04) [inline authentication/views.py:123]: See comment above.

File: observation_request/views.py [hunk 3]
```diff
@@ -38,10 +37,4 @@ def simple_request(request):
...
 @api_view(["POST"])
 def test_request(request):
-    parsed_data = parse_qs(request.body)
-    decoded_data = {
-        key.decode("utf-8"): value[0].decode("utf-8")
-        for key, value in parsed_data.items()
-    }
-
-    return HttpResponse(str(decoded_data))
+    return HttpResponse(request.data.dict())
```
Comment (btw04) [inline observation_request/views.py:40]: This seems unrelated

**Review by btw04 — CHANGES_REQUESTED** [review]
Agreed on merging this ASAP. Requested changes are minor

**Comment by btw04** [comment]
And please add the potential follow ups to the backlog on Trello. We might want a different category for these kind of "potentials" that aren't necessarily a fit for our backlog

File: observation_request/views.py (hunk 3 above)
Comment (bennetbo) [reply observation_request/views.py:40]: Now that we require login by default, accessing body breaks. That's why I changed it

**Review by btw04 — APPROVED** [review]
LGTM

**Pull request merged**